tests/e2e/run-scanner-v4.bats

```diff
@@ -398,7 +398,7 @@ EOF
     export SENSOR_SCANNER_SUPPORT=true
     # shellcheck disable=SC2030,SC2031
     export SENSOR_SCANNER_V4_SUPPORT=true
-    VERSION="${OPERATOR_VERSION_TAG}" deploy_stackrox_operator
+    CSV_VERSION="v${OPERATOR_VERSION_TAG}" deploy_stackrox_operator
```

Comment:
I just got to check this file hence this comment comes so late.

1. The use of `deploy_stackrox_operator` here and the use of `make -C operator deploy-previous-via-olm` below suggests this test file was never meant to run with midstream images and so by using `deploy_stackrox_operator` here we deceive ourselves that this _may_ work for P/Z whereas this actually doesn't. Would the tests still feel fine if we do the following?
```suggestion
    CSV_VERSION="v${OPERATOR_VERSION_TAG}" make -C operator deploy-via-olm
```
I think, consistency would be better than relying on the thin wrapper from `tests/e2e/lib.sh`.
Sorry, I know you did not touch _this_ aspect.

2. What makes me add comment here in the first place is again less than obvious mechanics of `CSV_VERSION` override. Would you be ok to swap it for `INDEX_IMG_TAG` and `INSTALL_VERSION`?

Reply:
The problem is that deploy_stackrox_operator does a couple more things (depends on the `kuttl` target and sets a couple of other env variables) and I _really_ don't think this is the right time to go and shave _that_ yak.

Reply:
Hm. Isn't there a problem that
* `deploy-previous-via-olm`
* `upgrade-via-olm`
* `upgrade-dirty-tag-via-olm`

all depend on `kuttl` but
* `deploy-via-olm`
* `deploy-dirty-tag-via-olm`

don't depend on it?

I mean the mere fact that I need to ask this and you need to spend time answering this is a sign to me that _that_ yak should have not been let grow hair from the beginning. From glancing at git history, I'm almost sure it was not intentional to have some targets depend on `kuttl` and others omit it but now I anyway have to guess why and waste your time on questions like this.

----

The absence of `REGISTRY_PASSWORD` and `REGISTRY_USERNAME` variables set in `Upgrade multi-namespace installation` test which calls `make -C operator deploy-previous-via-olm` directly makes me think that either tests are order-dependent and this test would fail should it be executed independently (or first in the batch) or that these variables are actually not needed for these scanner V4 tests.
Another yak that we let to happen?

----

There's no need to address these things in this PR but I would suggest you consider a follow-up to add `kuttl` as a dependency of `deploy-via-olm` and `deploy-dirty-tag-via-olm`.